packages/flutter/lib/src/gestures/multidrag.dart

```diff
@@ -231,6 +236,22 @@ abstract class MultiDragGestureRecognizer extends GestureRecognizer {
   /// [Drag] object returned by this callback.
   GestureMultiDragStartCallback? onStart;
 
+  /// Called when interaction starts. This limits the dragging behavior
```

Comment:
Can you put this doc in a template and reuse below?

Reply:
Yes, but how? I never did this. lol
Do you have any example/reference I can take a look?

Reply:
Thanks, done :)